packages/jest-snapshot/src/inline_snapshots.ts

```diff
@@ -8,49 +8,39 @@
 import fs from 'fs';
 import path from 'path';
 import semver from 'semver';
+import {loadPartialConfig} from '@babel/core';
+import {parse} from '@babel/parser';
 import {
   templateElement,
   templateLiteral,
   file,
   CallExpression,
 } from '@babel/types';
-import {Frame} from 'jest-message-util';
 
 import {Config} from '@jest/types';
 import {escapeBacktickString} from './utils';
+import type {Path} from 'types/Config';
 
-export type InlineSnapshot = {
-  snapshot: string;
-  frame: Frame;
-};
+export type InlineSnapshot = {|
```

Comment:
not valid TS 🙂

Reply:
Haha, yes I know, I'm actually much more familiar with typescript than flow so this change made me happy once I figured out what was going on. I just didn't notice that the file type had changed!